create_infotexts.py

```diff
@@ -81,9 +81,8 @@
 
 
 def load_places_mapping():
-    """
-    Reads wikitable html and returns a dictionary
-    :returns: dictionary with keywords as keys
+    """Read  wikitable html and return a dictionary
```

Comment:
For multi-line docstrings the initial line should be on the line following the `"""` (note that defining return/param counts as making it multiline).